common.py

```diff
@@ -50,6 +50,31 @@
         'swift-experimental-string-processing': 'swift/main',
         'swift-syntax': 'main',
     },
+    'release/5.10': {
+        'llvm-project': 'swift/release/5.10',
+        'swift-llvm-bindings': 'swift/release/5.10',
+        'swift': 'release/5.10',
+        'cmark': 'release/5.10',
+        'ninja': 'release',
+        'llbuild': 'release/5.10',
+        'swiftpm': 'release/5.10',
+        'swift-corelibs-libdispatch': 'release/5.10',
+        'swift-corelibs-foundation': 'release/5.10',
+        'swift-corelibs-xctest': 'release/5.10',
+        'swift-argument-parser': '1.2.2',
```

Comment:
Ah I didn't actually check these with update-checkout. Will do that now (just copied 5.9 and replaced with 5.10)

Reply:
Fixed